_i18n/en/resources/moneropedia/atomic-units.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: moneropedia
+entry: "Atomic Units"
+terms: ["atomic-units", "atomic-unit"]
+summary: "Atomic Units refer to the smallest fraction of 1 XMR according to the monerod implementation."
```

Comment:
Maybe the "according to the monerod implementation" should be dropped in the summary.